Makefile

```diff
@@ -61,10 +61,13 @@ doc-plot:
 doc:
 	make -C doc html-noplot
 
+.PHONY : doc-check
+doc-check:
+	make -C doc check
```

Comment:
Now that we are using "html-strict", shouldn't we remove this target from the parent Makefile?